server/src/main/java/org/opensearch/cluster/routing/allocation/DiskThresholdMonitor.java

```diff
@@ -356,6 +357,7 @@ public void onNewInfo(ClusterInfo info) {
         final Set<String> indicesToAutoRelease = StreamSupport.stream(state.routingTable().indicesRouting().spliterator(), false)
             .map(c -> c.key)
             .filter(index -> indicesNotToAutoRelease.contains(index) == false)
+            .filter(index -> !state.getMetadata().settings().get(IndexModule.INDEX_STORE_TYPE_SETTING.getKey()).equals(IndexModule.Type.REMOTE_SNAPSHOT.getSettingsKey()))
```

Comment:
@Bukhtawar I have added this so that disk threshold monitor filters out index backed by remote snapshot from list of indices to be released, but currently I can't find a way to test it locally (discussed more details on slack). Can you suggest a way to test this in run time (somehow the monitor selects some indices to auto release but not the remote ones)?

Reply:
Can we explore using a separate block so that concerns are separate and we don't run into each other?